utils/config.go

```diff
@@ -285,7 +285,7 @@ func InitAndLoadConfig(filename string) error {
 // LoadConfig will try to search around for the corresponding config file.
 // It will search /tmp/fileName then attempt ./config/fileName,
 // then ../config/fileName and last it will look at fileName
-func LoadConfig(fileName string) {
+func LoadConfig(fileName string) *model.Config {
```

Comment:
Adding the config as a return value with the goal of eliminating all side-effects of this function.